app/components/transfer/YoroiTransferStartPage.js

```diff
@@ -0,0 +1,84 @@
+// @flow
+import React, { Component } from 'react';
+import { observer } from 'mobx-react';
+import classnames from 'classnames';
+import { defineMessages, intlShape, FormattedHTMLMessage } from 'react-intl';
+import { Button } from 'react-polymorph/lib/components/Button';
+import { ButtonSkin } from 'react-polymorph/lib/skins/simple/ButtonSkin';
+
+import BorderedBox from '../widgets/BorderedBox';
+import styles from './YoroiTransferStartPage.scss';
+
+const messages = defineMessages({
+  title: {
+    id: 'yoroiTransfer.start.instructions.title',
+    defaultMessage: '!!!instruction',
+  },
+  text: {
+    id: 'yoroiTransfer.start.instructions.text',
+    defaultMessage: '!!!Transfer funds from another wallet.',
```

Comment:
Probably need to come up with something more descriptive to put here. In general through, I don't understand why you removed stuff from this page. The "Transfer from another wallet" feature will crash if you try and transfer before a wallet was created so you need to do the same as the Daedalus case where you ask users to create a wallet first.